doc/nrf/ug_partition_manager.rst

```diff
@@ -0,0 +1,189 @@
+.. _ug_pm:
+
+Partition Manager
+#################
+
+Partition Manager is a python script which sets the start address and size of all image partitions in a multi image build context.
```

Comment:
consider explanation on `image partition` term (glossary needed?)

Reply:
Something like this?

```
Partition Manager is a python script which sets the start address and size of all image partitions in a multi image build context.
An image partition is the flash area reserved for an image, to which the image is writter.
```

Reply:
almost:
```
Partition Manager is a python script which sets the start address and size of all image partitions in a multi image build context.
An image partition is the flash area reserved for an image, to which the image binary is written.
```